aviary/examples/external_subsystems/simple_weight/run_simple_weight.py

```diff
@@ -18,6 +18,9 @@
 # Allow for user overrides here
 prob.load_inputs('models/test_aircraft/aircraft_for_bench_FwFm.csv', phase_info)
 
+# Preprocess inputs
+prob.check_and_preprocess_inputs()
```

Comment:
good catch; we definitely need to always call this.

Reply:
Yes, any height-energy mission that doesn't run `check_and_preprocess_inputs()` will fail now because that's what we use to separate the phases into normal / reserve.

Reply:
Current Error Message if you use height_energy / solved_2DOF and don't call check-preprocess:

```
Traceback (most recent call last):
  File "/mnt/c/OMDAO/om-Aviary/aviary/examples/run_reserve_mission_cruise_fixedrange.py", line 61, in <module>
    prob.add_post_mission_systems()
  File "/mnt/c/OMDAO/om-Aviary/aviary/interface/methods_for_level2.py", line 1385, in add_post_mission_systems
    self.model.connect(f"traj.{self.regular_phases[0]}.timeseries.mass",
IndexError: list index out of range
```